product-features/farming/faqs.md

```diff
@@ -0,0 +1,118 @@
+
+</details>
+
+<details>
+
+<summary>Why is it more convenient to stake for longer periods?</summary>
+
+Because of the cooldown period. If you plan to stake for multiple cycles, it is more efficient to stake for the entire period upfront rather than withdrawing and restaking repeatedly.
+
+For example, if you want to farm for 12 cycles and choose to stake three times for 4-cycle periods, you will miss out on rewards for 3 cycles. This happens because each time you withdraw and restake, you enter a cooldown period. In contrast, if you stake directly for the full 12 cycles, you will only miss rewards for 1 cycle, the very first one.
+
+</details>
+
+<details>
+
+<summary>What can I do with my rewards?</summary>
+
+You have several options for your rewards: you can hold them, trade them, or generate compound interest. Compound interest occurs when your rewards generate more rewards. Here are some ways to achieve this within the ALEX Lab Platform:
+
+* Stake the rewards on [ALEX Staking](../stake.md). You can even buy LiALEX for auto-compounding rewards, maximizing your long-term yield.
+* Provide liquidity to a pool to earn a share of the trading fees. To further enhance your yield, stake the LP tokens in a farm.
+
+</details>
+
+<details>
+
+<summary>How can I restake my rewards in the farm?</summary>
+
+Reinvesting your rewards in the farm is an effective strategy to achieve compound interest, but it requires a few extra steps. To restake, you will need to transform your rewards into LP tokens first. There are two scenarios to consider.
+
+* **Case 1:** The reward token is one of the trading pair tokens.&#x20;
+* **Case 2:** The reward token is none of the trading pair tokens.
+
+Here are the steps to restake your farming rewards:
+
+1. [Swap](https://app.alexlab.co/swap) the rewards in order to obtain the liquidity pool trading pair tokens. This may involve one swap for Case 1 and two swaps for Case 2.
+2. [Provide liquidity](https://app.alexlab.co/pool) to the pool associated with the trading pair to receive LP tokens.
+3. Finally, stake the new LP tokens into the farm.
+
+</details>
+
+<details>
+
+<summary>What is APower?</summary>
+
+ALEX Staking Power, or APower, is a non-transferrable and non-tradable token. It is a special incentive that you can earn through staking on the ALEX Lab Platform, either by:
+
+1. **Stake $ALEX (1x Multiplier)**
+2. **Stake LP tokens through Yield Farming (0.3x Multiplier)**
+
+APower is the lottery ticket that allows you to take part in any future IDO rounds on our [Launchpad](https://app.alexlab.co/launchpad). There is no maximum amount of APower an address can earn over a period of time. If you are interested in frequently participating in IDOs, staking $ALEX would generate APower fastest.
+
+Every IDO is unique, however, and may have a cap on the amount of APower that can be allocated. This is to prevent IDOs dominated by a small group of "whale" members.
```

Comment:
Every IDO is unique, however, and may have a cap on the amount of APower that can be utilized to allocate IDO tickets. This prevents IDOs from being dominated by a small group of "whale" members.